setup.py

```diff
@@ -85,15 +85,11 @@ def make_parser(self, build_dir: str = ".") -> None:
 
 
 class BuildParser(MakeParser):
-    # pylint: disable=too-many-ancestors
-
```

Comment:
I think we should set the minimum required version to `pylint >=2.9.0`, as earlier versions will show an error because of the removed annotation.

Reply:
That's true. Changed in 0223d99.